### PR TITLE
Release announcement for Statefun 3.2.0

### PR DESCRIPTION
**Pull request by tillrohrmann**
This PR adds the release announcement for the Statefun 3.2.0 release.

cc @igalshilman for your input on the new improvements/features.

### CODE REVIEW

File: _posts/2022-01-27-release-statefun-3.2.0.md [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+- trohrmann:
+  name: "Till Rohrmann"
+  twitter: "stsffap"
+- igalshilman:
+  name: "Igal Shilman"
+  twitter: "IgalShilman"
+---
+
+Stateful Functions is a cross-platform stack for building Stateful Serverless applications, making it radically simpler to develop scalable, consistent, and elastic distributed applications.
+This new release brings various improvements to the StateFun runtime, a leaner way to specify StateFun module components, and a brand new GoLang SDK!
```
Comment (sjwiesman) [inline _posts/2022-01-27-release-statefun-3.2.0.md]: s/golang/javascript/g

Comment (tillrohrmann) [reply _posts/2022-01-27-release-statefun-3.2.0.md]: Good catch. Will correct it.

File: _posts/2022-01-27-release-statefun-3.2.0.md [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+  - A unified cross-language way to send, receive and store values across languages.
+  - Dynamic `ValueSpec` to describe the state name, type, and possibly expiration configuration at runtime.
+
+You can get started by adding the SDK to your project.
+
+`npm install apache-flink-statefun@3.2.0`
+
+For a detailed SDK tutorial, we would like to encourage you to visit:
+
+  - [JavaSCript SDK Documentation]({{site.DOCS_BASE_URL}}flink-statefun-docs-release-3.2/docs/sdk/js/)
```
Comment (igalshilman) [inline _posts/2022-01-27-release-statefun-3.2.0.md]: typo: It should say JavaScript

File: _posts/2022-01-27-release-statefun-3.2.0.md [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+For more details, check the complete [release changelog](https://issues.apache.org/jira/secure/ReleaseNote.jspa?projectId=12315522&version=12350540)
+and the [updated documentation]({{site.DOCS_BASE_URL}}flink-statefun-docs-release-3.2/).
+We encourage you to download the release and share your feedback with the community through the [Flink mailing lists](https://flink.apache.org/community.html#mailing-lists)
+or [JIRA](https://issues.apache.org/jira/browse/)!
+
+{% toc %}
+
+## New Features
+
+### A brand new JavaScript SDK
```
Comment (igalshilman) [inline _posts/2022-01-27-release-statefun-3.2.0.md]: Perhaps we should say that this is for NodeJs in particular? 
Something like
`A brand new JavaScript SDK for NodeJS`

**Review by igalshilman — APPROVED** [review]
Thanks @tillrohrmann, the announcement looks really nice :-)

File: _posts/2022-01-27-release-statefun-3.2.0.md (hunk 2 above)
Comment (tillrohrmann) [reply _posts/2022-01-27-release-statefun-3.2.0.md]: Indeed. Will correct it.

File: _posts/2022-01-27-release-statefun-3.2.0.md (hunk 3 above)
Comment (tillrohrmann) [reply _posts/2022-01-27-release-statefun-3.2.0.md]: This is better. I will change it.

**Comment by tillrohrmann** [comment]
Thanks for the review @igalshilman. I've addressed your comments and pushed a fix up commit.

**Comment by tillrohrmann** [comment]
Manually merged via 4d2acc8fc1a001794ab54201a2f98cf33a16cdb1

**Pull request closed without merging**